es-ds-docs/pages/atoms/layout.vue

```diff
@@ -1,5 +1,328 @@
 <template>
     <div>
-        atoms - layout
+        <h1>
+            Layout
+        </h1>
+        <p>
+            Extended from 
+            <NuxtLink to="https://bootstrap-vue.org/docs/components/layout" target="_blank">
+                bootstrap-vue layout
+            </NuxtLink>
+        </p>
+        <h2 class="mt-200">
+            Responsive Breakpoints
```

Comment:
Nit: this is how it was before, but can we follow our own [Typography guidelines](https://design.energysage.dev/atoms/typography) and make this sentence case like "Responsive breakpoints"?